src/Featurizers/ImputationMarkerFeaturizer.h

```diff
@@ -11,20 +11,31 @@ namespace Microsoft {
 namespace Featurizer {
 namespace Featurizers {
 
+/////////////////////////////////////////////////////////////////////////
+///  \class        ImputationMarkerTraits
+///  \brief         Traits for mapping the input/output types for the ImputationMarker.
+///                 This allows us to only change one place if typings need to change.
+///
+template <typename TransformedT>
+struct ImputationMarkerTraits{
```

Comment:
>ImputationMarkerTraits [](start = 7, length = 22)

Should this be generalized since the same code is used at least in two places?

Reply:
I am not sure if the input/output type mappings will be the same for each featurizer which is why I have them separate for each one. Currently that is the case, just not sure about always. @davidbrownellWork do you know?

Reply:
I moved this code into a general location and have update the transformers to they use that one.